pkg/typesystem/typesystem.go

```diff
@@ -227,6 +229,10 @@ func New(model *openfgav1.AuthorizationModel) (*TypeSystem, error) {
 		}
 	}
 
+	wgb := graph.NewWeightedAuthorizationModelGraphBuilder()
+	// TODO: this will require a deprecation not ignore the error and remove nil checks
+	weightedGraph, _ := wgb.Build(model)
```

Comment:
should we store the error (not return it) so that if there was an error we do _not_ use fast path v2?

Reply:
ah never mind, i guess this is why you added

```go
if t.authzWeightedGraph == nil {
		return false
	}
```

down below.